src/txmempool.cpp

```diff
@@ -98,6 +100,27 @@ void CTxMemPool::UpdateForDescendants(txiter updateIt, cacheMap &cachedDescendan
     mapTx.modify(updateIt, update_descendant_state(modifySize, modifyFee, modifyCount));
 }
 
+void CTxMemPool::GetRebroadcastTransactions(std::set<uint256>& setRebroadcastTxs)
+{
+    // Don't rebroadcast txns during importing, reindex, or IBD to ensure we don't
+    // accidentally spam our peers with old transactions.
+    if (::ChainstateActive().IsInitialBlockDownload() || ::fImporting || ::fReindex) return;
+
+    BlockAssembler::Options options;
+    options.nBlockMaxWeight = MAX_REBROADCAST_WEIGHT;
+    options.m_max_tx_time = std::chrono::seconds(GetTime()) - REBROADCAST_MIN_TX_AGE;
```

Comment:
91eefef24 isn't compiling for me due to `no member named 'm_max_tx_time' in 'BlockAssembler::Options'`

Reply:
fixed. all commits should now build.